examples/Advanced_Sounding_With_Complex_Layout.py

```diff
@@ -87,83 +76,67 @@
 ###########################################
```

Comment:
Let's go ahead and remove this line of pound signs (hashtags) so that the text below will appear as a code comment instead of as plain text on the webpage. Additionally, I think it would be more readable to add make in some line spaces in the code between each comment that you have. It shouldn't affect the creation of the graphic like we had previously. That was coming as a result of the full line of pound signs. So for example, adding back in a blank line before line 84, 88, 91, etc.